cli.js

```diff
@@ -52,15 +54,21 @@ const cli = meow(`
   }
 });
 
+if (cli.input.length == 0)
+  cli.showHelp(1);
 
 const conversions = cli.input.map(function(conversion) {
 
-  const delimiter = conversion.includes(path.delimiter) ? path.delimiter : ':';
+  const hasDelimiter = conversion.includes(path.delimiter);
+  if (!hasDelimiter) {
+     console.error(error(`  no <diagramFile>{path.delimiter}<outputConfig> param provided`));
```

Comment:
```suggestion
     console.error(error(`Error: no <diagramFile>{path.delimiter}<outputConfig> param provided`));
```

Reply:
Maybe we could add here the prefix to make it just a little clearer. Just an idea, the color should do the trick as well.

Reply:
Sure. I added the whitespace to align with help indentation, should I keep it?

Reply:
Oh sorry, yes of course!